_posts/2015-11-17-we-have-been-busy.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "We. Have. Been. Busy!"
+author: Jan
+layout: blog-post-tgif
+comments: true
+---
+
+*phew* what a summer (and fall)! The Hoodie community has been very busy, and at the same time, we’ve neglected the blog a little bit. This will be fixed now and we’ll start with getting you up to speed what has happened in the past <strike>three</strike> five months.
+
+This is a mini-series that we’ll publish every Tuesday for the next four weeks. We’ll start with this brief overview and and then, in each issue, go into details about what happened.
+
+*If you want to get involved with the publication process on the Hoodie blog, or the project in general: [we are actively recruiting and editorial team](https://github.com/hoodiehq/discussion/issues/84).*
+
+Here are the highlights
+
+## 1. Hoodie 1.0 milestones closing in
+
+We made some **significant progress** on [the Hoodie 1.0 milestones](http://gr2m.github.io/milestones/). Most notably, we [completely re-structured how Hoodie is put together, and tested and released](https://github.com/hoodiehq/discussion/issues/76). This work is already available in our new @next tag on npm, albeit not 100% functional yet. If you are keen on giving it a spin and help fix missing features, run *npm install hoodie@next* to get the work-in-progress version. See [https://github.com/hoodiehq/hoodie](https://github.com/hoodiehq/hoodie) for more details and a a way to find and file issues. The main work is done in the [Client](https://github.com/hoodiehq/hoodie-client), [Server](https://github.com/hoodiehq/hoodie-server) and [Admin Dashboard](https://github.com/hoodiehq/hoodie-admin-dashboard) modules.
```

Comment:
@Charlotteis we could link to https://github.com/hoodiehq/hoodie-server/tree/next would that help?